e2e/blockstorage/resource_block_storage.go

```diff
@@ -229,26 +294,26 @@ func resourceExistsBlockStorage(d *schema.ResourceData, m interface{}) (bool, er
 	return true, nil
 }
 
-func calculateIOPS(size float64) int {
+func calculateIOPS(size float64) string {
 	switch size {
```

Comment:
You can simply use a multiplication factor of 15, just multiply it with size and you have the IOPS.